civis/io/_tables.py

```diff
@@ -115,6 +127,23 @@ def read_civis(table, database, columns=None, use_pandas=False,
     if client is None:
         # Instantiate client here in case users provide a (deprecated) api_key
         client = APIClient(api_key=api_key, resources='all')
+
+    if use_pandas == True:
+        if dtype == 'redshift':
+            if NO_NUMPY:
+                raise ImportError("`dtype='redshift'` specified but numpy not installed.")
```

Comment:
Is this `numpy` check needed? If we have `pandas`, then we know that we have `numpy`.